factcast-store-pgsql/README.md

```diff
@@ -1,5 +1,14 @@
-### factcast-store-pgsql
+# factcast-store-pgsql
 
 PostgreSQL based implementation of a FactStore.  
 
 Uses LISTEN / NOTIFY in order to query with minimum latency and relies on JSONB Columns for indexing/filtering. 
+
+# Schema registries
+
+The following protocols exist for reading schemata: 
+
+- http://...
```

Comment:
plz also add to factcast-site/documentation/content/setup/properties

Reply:
Done, please check.